erpnext/accounts/doctype/payment_entry/payment_entry.py

```diff
@@ -1172,3 +1172,20 @@ def update_item(source_doc, target_doc, source_parent):
 	}, target_doc, set_missing_values)
 
 	return doclist
+
+
+@frappe.whitelist()
+def init_print_cheque(start, selected_docs, doctype):
+	assigned_doc = []
+	series = int(start)
+	docs = json.loads(selected_docs)
+	for doc in docs:
+		if not frappe.db.get_value(doctype, doc.get("name"), "reference_no"):
+			frappe.db.set_value(doctype, doc.get("name"), {
+				"reference_no": series,
+				"reference_date": nowdate()
+			})
+			assigned_doc.append({"name": frappe.utils.get_link_to_form(
+				doctype, doc.get("name")), "series": series})
```

Comment:
Add the title as the label of the message

```suggestion
			assigned_doc.append({
				"name": frappe.utils.get_link_to_form(doctype, doc.get("name"), doc.get("title")),
				"series": series
			})
```